packages/notifi-react/lib/context/NotifiHistoryContext.tsx

```diff
@@ -174,6 +179,14 @@ export const NotifiHistoryContextProvider: FC<
     }
   }, [frontendClientStatus, getHistoryItems, cardConfig]);
 
+  useEffect(() => {
+    // reload history items with unread filter toggle change
```

Comment:
I would suggest to change the comment because in `notifi-react` there is no `filter toggle change` concept.
We might want to make it more neutral like
```ts
// NOTE: refresh history while switch between the following modes: only showing unread (`isIncludeRead=false`) & showing all histories (`isIncludeRead=ture`)